File: src/FirstBundle/Entity/Classe.php
<?php

namespace FirstBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Classe
 *
 * @ORM\Table(name="class")
 * @ORM\Entity
 */
class Classe
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="name", type="string", length=255, nullable=false)
     */
    private $name;

    /**
     * @var integer
     *
     * @ORM\Column(name="nb_child", type="integer", nullable=false)
     */
    private $nbChild;

    /**
     * @var integer
     *
     * @ORM\Column(name="level", type="integer", nullable=false)
     */
    private $level;

    /**
     * @var string
     *
     * @ORM\Column(name="teacher_name", type="string", length=255, nullable=false)
     */
    private $teacherName;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set name
     *
     * @param string $name
     *
     * @return Classe
     */
    public function setName($name)
    {
        $this->name = $name;

        return $this;
    }

    /**
     * Get name
     *
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * Set nbchild
     *
     * @param integer $nbChild
     *
     * @return Classe
     */
    public function setNbchild($nbChild)
    {
        $this->nbChild = $nbChild;

        return $this;
    }

    /**
     * Get nbchild
     *
     * @return int
     */
    public function getNbchild()
    {
        return $this->nbChild;
    }

    /**
     * Set level
     *
     * @param integer $level
     *
     * @return Classe
     */
    public function setLevel($level)
    {
        $this->level = $level;

        return $this;
    }

    /**
     * Get level
     *
     * @return int
     */
    public function getLevel()
    {
        return $this->level;
    }

    /**
     * Set teacher
     *
     * @param string $teacher
     *
     * @return Classe
     */
    public function setTeacher($teacher)
    {
        $this->teacher = $teacher;

        return $this;
    }

    /**
     * Get teacher
     *
     * @return string
     */
    public function getTeacher()
    {
        return $this->teacher;
    }

}

File: Pimobile/src/com/mycompany/entities/Classe.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package com.mycompany.entities;

/**
 *
 * @author Aziz
 */
public class Classe {
    private int id;
    private String name;
    private  int nbchild;
    private  int level;
    private String teacher;

    public Classe() {
    }

    public Classe(int id, int nbchild, int level, String teacher) {
        this.id = id;
        this.nbchild = nbchild;
        this.level = level;
        this.teacher = teacher;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getNbchild() {
        return nbchild;
    }

    public void setNbchild(int nbchild) {
        this.nbchild = nbchild;
    }

    public int getLevel() {
        return level;
    }

    public void setLevel(int level) {
        this.level = level;
    }

    public String getTeacher() {
        return teacher;
    }

    public void setTeacher(String teacher) {
        this.teacher = teacher;
    }
    
}
